TourMate/TourMate/Controllers/AuthenticationController.swift

```diff
@@ -19,9 +20,17 @@ struct AuthenticationController {
 
         let (hasRegistered, errorMessage) = await authenticationManager.registerUser(email: email,
                                                                                      password: password)
-        return (hasRegistered, errorMessage)
+        guard hasRegistered else {
+            return (hasRegistered, errorMessage)
+        }
+
+        await logIn(email: email, password: password)
```

Comment:
I think Firebase's `createUser` will sign in the user after creation is successful. Maybe we can discuss the flow on authentication + user persistence 

![Screenshot 2022-03-14 at 11 19 01](https://user-images.githubusercontent.com/45381509/158099046-e5d58286-44d5-4c05-ad93-f6293979510a.png)
.

Reply:
Oh ok nice, was not aware of that. Will remove it later